scylla-proxy/src/actions.rs

```diff
@@ -409,67 +409,67 @@ impl RequestReaction {
 
 struct ExampleDbErrors;
 impl ExampleDbErrors {
-    pub fn syntax_error() -> DbError {
+    fn syntax_error() -> DbError {
         DbError::SyntaxError
     }
```

Comment:
This `struct` should be `pub` instead; it's intended as a public API even though not yet used by any external code.

Reply:
Done